### DISCUSSION TITLE
Web app version?

### DISCUSSION THREAD

**Discussion by ithakaa** [athensresearch/athens#1380] [Community Call]
Loving Athens

Is there a web based version I can self host ?

Would be great to be able to use it on remote systems.

**Comment by sawhney17** [comment]
Currently, there is no web based version that you can self host although you can manually trigger a demo web build which will however not be able to persist changes and cannot(to my knowledge) be used with an index.transit. Luckily a self hosted web version is in the roadmap is a priority.

**Comment by solarkraft** [comment]
A web app would be the ultimate compatibility solution for systems without an Athens client and would massively alleviate the pain of not having a mobile app (#954). 

Unfortunately it may not be as much of a no-brainer as I had initially thought: The [web demo](https://athensresearch.github.io/athens/) doesn't start on iOS, presumably due to the use of web workers. 

Eventually (not intimately knowing the Athens architecture), the web client may become the same as the front-end portion of the desktop app, connecting to the RTC backend.

**Comment by solarkraft** [comment]
I see it's [on the roadmap](https://github.com/athensresearch/athens/issues/842), listed as a short-term goal (before a mobile app).
